packages/cli/src/commands/generate/sdl/sdl.js

```diff
@@ -242,7 +242,7 @@ export const builder = (yargs) => {
     .option('tests', {
       description: 'Generate test files',
       type: 'boolean',
-      default: false,
+      // don't give it a default value, it gets overwritten in  first few lines of the handler
```

Comment:
This is the key change

Reply:
Oh, I wonder if I put `default: false,` on the wrong line. And intended it for the docs section.